scene/gui/item_list.cpp

```diff
@@ -28,8 +28,10 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 #include "item_list.h"
+#include "os/input.h"
 #include "os/os.h"
 #include "project_settings.h"
+#include <joypad.h>
```

Comment:
Godot headers should be included with quotes. `< >` is for system headers.

Reply:
@akien-mga done